lib/index.js

```diff
@@ -442,6 +454,10 @@ Builder.prototype.build = function(options) {
 
 			// Compile theme if not cached or RTL is requested and missing in cache
 			if (!themeCache || (options.rtl && !themeCache.result.cssRtl)) {
+				var scopeOptions = options.scope;
+				if (scopeOptions.selector && scopeOptions.embeddedFilePath && scopeOptions.embeddedCompareFilePath && scopeOptions.baseFile) {
```

Comment:
This check also needs to be done in the `else` clause below, where currently only `readDotTheming(dotThemingInputPath).then(addInlineParameters).then(that.cacheTheme.bind(that));` is called (line 467)

Reply:
Done!